lib/main.dart

```diff
@@ -11,17 +13,26 @@ void main() {
   WidgetsFlutterBinding.ensureInitialized();
   BlocSupervisor.delegate = SimpleBlocDelegate();
   final UserRepository userRepository = UserRepository();
+  final GroupsRepository groupsRepository = GroupsRepository();
 
-  runApp(
-    BlocProvider(
-      create: (context) =>
-          AuthenticationBloc(userRepository: userRepository)..add(AppStarted()),
-      child: RepositoryProvider<UserRepository>(
-        create: (context) => userRepository,
-        child: MyApp(),
+  runApp(MultiBlocProvider(
+    providers: [
+      BlocProvider(
+        create: (context) => AuthenticationBloc(userRepository: userRepository)
+          ..add(AppStarted()),
+        child: RepositoryProvider<UserRepository>(
+          create: (context) => userRepository,
+          child: MyApp(),
+        ),
       ),
-    ),
-  );
+      BlocProvider(
```

Comment:
We don't need to create and pass the Group bloc as we only need it in the Groups Screen

Reply:
okay commenting it for now because might be useful when need something from groups  list